packages/app/src/sections/dashboard/Stake/StakingPortfolio.tsx

```diff
@@ -1,145 +1,102 @@
-import { truncateNumbers } from '@kwenta/sdk/utils'
-import { useRouter } from 'next/router'
 import { FC } from 'react'
 import { useTranslation } from 'react-i18next'
 import styled from 'styled-components'
 
 import Button from 'components/Button/Button'
-import { FlexDivRowCentered } from 'components/layout/flex'
+import { FlexDivCol, FlexDivRow, FlexDivRowCentered } from 'components/layout/flex'
+import { Body, Heading } from 'components/Text'
 import { EXTERNAL_LINKS } from 'constants/links'
-import ROUTES from 'constants/routes'
-import { SplitStakingCard } from 'sections/dashboard/Stake/card'
-import { Heading } from 'sections/earn/text'
-import { useAppSelector } from 'state/hooks'
-import {
-	selectClaimableBalance,
-	selectEscrowedKwentaBalance,
-	selectKwentaBalance,
-	selectStakedEscrowedKwentaBalance,
-	selectStakedKwentaBalance,
-	selectTotalVestable,
-} from 'state/staking/selectors'
+import { StakingCards } from 'pages/dashboard/staking'
 import media from 'styles/media'
 
 export enum StakeTab {
 	Staking = 'staking',
 	Escrow = 'escrow',
-	TradingRewards = 'trading-rewards',
-	Redemption = 'redemption',
+	Redeem = 'redeem',
 }
 
 type StakingPortfolioProps = {
-	setCurrentTab(tab: StakeTab): void
+	cards: StakingCards[]
 }
 
-const StakingPortfolio: FC<StakingPortfolioProps> = ({ setCurrentTab }) => {
+const StakingPortfolio: FC<StakingPortfolioProps> = ({ cards }) => {
```

Comment:
Will probably be worth wrapping this component in memo since cards prop will be memoized

Reply:
Good idea. Updated.